docs/content.md

```diff
@@ -442,3 +445,64 @@ When updating not just the text content within the file, but the actual list of
 - Export the XLSX to CSV with filename `internship_providers.csv`
 - Place it in the `lib/tasks/support` directory.
 - Run `bundle exec rake teaching_internship_providers:generate`.
+
+
+## Creating a new page
+
+1. **Make sure you start by creating a new branch on master**
+2. Navigate to the right hand side bar, where you want the new page to sit
```

Comment:
Do we need to be more specific as there are the sections that have the blue arrows and there are also the red <> components. So I wouldn't know which one I was meant to be setting it up in or what the difference is between the two if they both have the same name?

Reply:
@MaxineCl can add it. Is it that the blue arrows are the pages, the reds are components?